index.html

```diff
@@ -3879,8 +3940,7 @@ <h2 id="GainNode">
           </dt>
           <dd>
             <p>
-              Represents the amount of gain to apply. Its default
-              <code>value</code> is 1 (no gain change). This parameter is
```

Comment:
I agree. I thought about keeping them, but having the same info in two locations is also problematic. I can revert part of this commit.